manage.py

```diff
@@ -5,6 +5,11 @@
 if __name__ == "__main__":
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "evap.settings")
 
+    if "test" in sys.argv:
+        from typeguard import install_import_hook
+
+        install_import_hook(("evap", "tools"))
+
```

Comment:
We have an implementation of "are we testing?" in `settings.py` and also have a block of things we do in this case: https://github.com/e-valuation/EvaP/blob/cdbdc9d632413f0e8ecdf3cfceb23cdd9eb86a29/evap/settings.py#L419-L422

Could this stuff also be done there?